loglist/logfilter.go

```diff
@@ -45,11 +49,18 @@ func (ll *LogList) Compatible(rootedChain []*x509.Certificate, roots LogRoots) L
 		return compatible
 	}
 	for _, l := range ll.Logs {
-		// Check root is accepted.
-		for _, r := range roots[l.URL] {
-			if r.Equal(rootedChain[len(rootedChain)-1]) {
-				compatible.Logs = append(compatible.Logs, l)
-				break
+		// If root set is not set, we treat Log as compatible assuming no
+		// knowledge on its roots.
```

Comment:
nit: (sorry) 'no knowledge of its roots' is probably what you mean?

Reply:
Fixed.